### PR TITLE
New launch files for PRL+ 80

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

File: cob_bringup/robots/cob3-7.xml [hunk 1]
```diff
@@ -101,15 +100,15 @@
 			<arg name="start_manager" value="false"/>
 			<arg name="nodelet_manager" value="cam3d_nodelet_manager"/>
 		</include-->
-		<include file="$(find cob_sound)/ros/launch/sound.launch" />
+		<!--include file="$(find cob_sound)/ros/launch/sound.launch" /-->
```
Comment (floweisshardt) [inline cob_bringup/robots/cob3-7.xml]: why is sound deactivated?

File: cob_bringup/robots/cob3-7.xml [hunk 2]
```diff
@@ -101,15 +100,15 @@
...
 
-		<include file="$(find cob_bringup)/components/light.launch" >
+		<!--include file="$(find cob_bringup)/components/light.launch" >
```
Comment (floweisshardt) [inline cob_bringup/robots/cob3-7.xml]: why is light deactivated?

File: cob_hardware_config/cob3-7/config/torso_canopenmaster.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+devices:
+  - name: /dev/pcan2
+    baudrate: 500K
+    sync_interval: 10
+# - name: /dev/pcan32
+#   baudrate: 1M
+#   sync_interval: 5
+chains: ["torso_controller"]
```
Comment (floweisshardt) [inline cob_hardware_config/cob3-7/config/torso_canopenmaster.yaml:8]: why do we need two canopenmaster.yaml files (for tray and torso)? isn't it possible to integrate all canopen devices and chains on one pc into one canopenmaster?

Comment (NadiaHG) [inline cob_hardware_config/cob3-7/config/torso_canopenmaster.yaml:8]: It is not tested, but then, should we have torso_vX.launch , tray_vX.launch and canopen.launch?

**Pull request closed without merging**